ui/article-renderer/src/main/java/org/cru/godtools/article/fragment/ArticlesFragment.java

```diff
@@ -0,0 +1,68 @@
+package org.cru.godtools.article.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.cru.godtools.article.R;
+import org.cru.godtools.article.adapter.ArticleAdapter;
+import org.cru.godtools.articles.aem.view_model.ArticleViewModel;
+
+import java.util.Objects;
+
+public class ArticlesFragment extends Fragment {
+    private static final String MANIFEST_Key = "manifest-key";
+    private ArticleAdapter mAdapter;
+
+    public static ArticlesFragment newInstance(String manifestKey) {
+        ArticlesFragment fragment = new ArticlesFragment();
+        Bundle args = new Bundle();
+        args.putString(MANIFEST_Key, manifestKey);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (!(context instanceof ArticleAdapter.Callback)) {
+            throw new IllegalArgumentException("Callback not integrated");
+        }
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        super.onCreateView(inflater, container, savedInstanceState);
+
+        return inflater.inflate(R.layout.fragment_articles, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        RecyclerView recyclerView = Objects.requireNonNull(getActivity())
```

Comment:
try to limit the amount of random code initialization that happens within the various lifecycle methods. you should organize the logic into helper methods like `setupRecyclerView()`. This helps keep fragments and activities maintainable long term by reducing complexity of the lifecycle event methods.

Reply:
Also, you should utilize Butterknife to bind views you need within fragments and activities. Extend `BaseFragment` and see https://github.com/CruGlobal/godtools-android/blob/develop/ui/article-renderer/src/main/java/org/cru/godtools/article/fragment/CategoriesFragment.java#L27-L29 for how to simply get the `RecyclerView`